### PR TITLE
Remove Netlify Site API data dependency

### PR DESCRIPTION
**Pull request by skogsmaskin**
Since the Netlify site API returned 401s when asking for a Site deployed from a private repository, we need to rely solely on config.

The config has chaged like this:

What previously was `site.name` is now the actual Netlify name for the site (i.e. `sanity-gatsby-blog-20-studio`)

What was `site.name` before is not `site.title` (custom overridable title for the site).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Sanity Studio Dashboard Widget for triggering Netlify builds.
 
-## Installing
+rea## Installing
```
Comment (mariuslundgard) [inline README.md]: typo?

File: src/props.ts [hunk 2]
```diff
@@ -16,13 +15,17 @@ const INITIAL_PROPS = {
...
-    mergeMap(fetchSite),
-    toArray()
-  )
+  const sites = (options.sites || []).map(site => ({
+    id: site.siteId,
+    name: site.name,
+    title: site.title,
+    buildHookId: site.buildHookId,
+    url: site.name && `${site.hostname}`,
+    adminUrl: site.name && `${site.hostname}`,
```
Comment (mariuslundgard) [inline src/props.ts]: Shouldn't we remove `adminUrl` now?

Comment (kmelve) [reply src/props.ts]: Isn't it good to have it there? Since we can derive it from the siteId?

Comment (skogsmaskin) [reply src/props.ts]: If there was a need before, nothing is stopping us from still having it now too.

File: README.md (hunk 1 above)
Comment (skogsmaskin) [reply README.md]: Yeah

File: src/props.ts (hunk 2 above)
Comment (skogsmaskin) [reply src/props.ts]: > Seems like it's not quite done?

There were some issues yes, should be sorted out now.

**Pull request merged**